rom/dev/src/main.rs

```diff
@@ -378,3 +363,36 @@ fn panic_is_possible() {
     // The existence of this symbol is used to inform test_panic_missing
     // that panics are possible. Do not remove or rename this symbol.
 }
+
+#[inline(always)]
+fn validate_trng_config(env: &mut RomEnv) {
+    if !cfi_launder(env.soc_ifc.debug_locked())
```

Comment:
We don't want to have these sorts of single branches. Nick wanted the only branches to be branches to cfi_panic

Reply:
Addressed by my push.